src/help.raw

```diff
@@ -84,7 +84,7 @@ OPTIONS
                            can be used multiple times to add multiple paths.
  -M, --menu-font FONT      Use FONT for the font in menus.
  -B, --image-bg STYLE      Set background for transparent images and the like.
-                           Accepted values: white, black, default
+                           Accepted values: default, checks, or a XColor (eg. #428bdd)
```

Comment:
Old code accepted `"checks"`, `"white"` and `"black"` (contrary to the manual), the new one actually does accept `"default"` - if it wasn't before, maybe leave it out here and default to `"black"` as mentioned below?

Reply:
The old code worked like that:

| | "white"  | "black" | "checks" | [anything else] |
| ------------- | ------------- | ------------- | ------------- | ------------- |
| Windowed  | White  | Black | Checks | Checks |
| Fullscreen  | White  | Black | Checks | Black |

The new code works the same, but [anything else] is now either no `--image-bg` at all, or `--image-bg default`. Any other string is converted to a color, which is then used for both windowed and fullscreen mode.

Reply:
Okay, then never mind my comments, looking great.